test/descriptive-link-text.md

```diff
@@ -0,0 +1,23 @@
+# Descriptive link text
```

Comment:
Please add another test file to confirm that setting the banned list to `[]` allows the default banned items - in other words, makes the rule allow everything.